sp800-63b/sec5_authenticators.md

```diff
@@ -341,7 +341,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 If matching is performed centrally:
 
 * Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
-* Biometric revocation SHOULD be implemented.
+* Biometric revocation SHALL be implemented. Referred to as biometric template protection in [ISO/IEC 24745](#ISO24745).
```

Comment:
Second sentence has no verb. How about, "Biometric revocation, referred to as biometric template protection in [ISO/IEC 24745](#ISO24745), SHALL be implemented."